app/views/admin/infobase/_orders.html.haml

```diff
@@ -0,0 +1,17 @@
+.ui.top.attached.tabular.menu
+  %a.item{"data-tab" => "Monday"} Monday
+  %a.item{"data-tab" => "Tuesday"} Tuesday
+  %a.item{"data-tab" => "Wednesday"} Wednesday
+  %a.item{"data-tab" => "Thursday"} Thursday
+  %a.item{"data-tab" => "Friday"} Friday
+
+.ui.bottom.attached.tab.segment{"data-tab" => "Monday", style: 'height: 630px; overflow: auto'}
+  = render 'order_form', orders: @orders.where("strftime('%w', updated_at) = ?", '1')
```

Comment:
1. Too much logic on the view
2. Loop
3. I, NO JOKES, struggling to understand what this line does `@orders.where("strftime('%w', updated_at) = ?", '1')` 
I believe it SOMEHOW takes orders by week days, but that looks awful. Please consider refactoring, I'll take a look later